packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -156,6 +162,11 @@ export default function HeaderEditMode() {
 		window?.__experimentalEnableZoomedOutView && isVisualMode;
 	const isZoomedOutView = blockEditorMode === 'zoom-out';
 
+	const slideY = {
+		hidden: { y: '-50px' },
+		hover: { y: 0, transition: { type: 'tween', delay: 0.2 } },
+	};
```

Comment:
If this object is non-dynamic, then you could put it outside the component to get the benefits of referential equality.

Reply:
yea good call, I think this is the case in other places as well